docker/tests/containerTestCommon.psm1

```diff
@@ -213,6 +213,56 @@ function Get-ContainerPowerShellVersion
     return (Get-Content -Encoding Ascii $testContext.resolvedLogPath)[0]
 }
 
+
+function Get-DefaultPreviewConfigForPackageValidation
+{
+    @{  'centos7'='rhel.7';
```

Comment:
Add a comment about where to get the list of the supported OS for each release

Reply:
Good point. Updated.